### ISSUE TITLE
Add more cocktails

### ISSUE THREAD

**Issue by tmcw** [tmcw/flair#3]
https://archive.org/stream/modernamericandr00kapp/modernamericandr00kapp_djvu.txt is public domain and classy, but very outmoded.

**Comment by tmcw** [comment]
opendrinks has 370 recipes that are structured, but pretty inconsistent in formatting. https://github.com/alfg/opendrinks

**Comment by tmcw** [comment]
In the meantime, I've been mostly pulling from Wikipedia, which in turn pulls from the 'official' IBA list.

**Comment by tmcw** [comment]
Reviewing wikipedia and some popular ones that aren't there yet:

- https://en.wikipedia.org/wiki/Margarita
- https://en.wikipedia.org/wiki/White_Russian_(cocktail)
- https://en.wikipedia.org/wiki/Gimlet_(cocktail)
- https://en.wikipedia.org/wiki/Mint_julep
- https://en.wikipedia.org/wiki/Tequila_Sunrise_(cocktail)
- https://en.wikipedia.org/wiki/Paloma_(cocktail)
- https://en.wikipedia.org/wiki/Greyhound_(cocktail)

**Comment by wooorm** [comment]
Any reason why you’re not pulling from the “official” IBA directly? It has _relatively_ structured data: https://iba-world.com/new-era-drinks/

**Comment by tmcw** [comment]
Nope, that'd be a good idea! It'll require a scrape pass & then moderate intelligent parsing for ingredients and and types of glasses to fit the data model.

**Comment by wooorm** [comment]
OK, scraped unforgettables and new era’s, but https://iba-world.com/contemporary-classics/ is an entirely different format 😭

**Comment by wooorm** [comment]
Alright well that took longer than expected. Walked through all existing, and added all others from iba. Now to add icons for new glasses. Expect them tonight or tomorrow!

**Comment by tmcw** [comment]
Thanks to @wooorm the list is now expansive and awesome!

There are some drinks that are not iba-certified but still commonplace that we can still add - this afternoon might dig up the 'tonic' (g&t, vodka tonic, etc) family

**Comment by wooorm** [comment]
Some great recipes can also be found on [imbibe](https://imbibemagazine.com/category/recipes/) and [punch](https://punchdrink.com/recipes/)!

**Comment by tmcw** [comment]
Realizing that we're pretty low on tiki drinks! Via [gastropod's tiki time episode](https://gastropod.com/tiki-time/), via [leanne](https://twitter.com/LeanneFaybraham), there's [Jeff Berry](https://beachbumberry.com/index.html) who decoded a lot of them and has classy nice versions.
